frontend/src/app/shared/components/resizer/resizer/wp-resizer.component.ts

```diff
@@ -222,4 +218,8 @@ export class WpResizerDirective extends UntilDestroyedMixin implements OnInit, A
       this.resizer.classList.remove('-error-font');
     }
   }
+
+  private setWidthVariable(value:number):void {
+    document.documentElement.style.setProperty("--split-screen-width", `${value}px`);
```

Comment:
💡 this turns the resizer component, that before aimed to be reusable, to be specific to the split view. This could be avoided by passing in the variable to alter.

Reply:
Done 👍